docs/howto/garbage-collection/index.md

```diff
@@ -238,7 +238,7 @@ spark.hadoop.lakefs.gc.do_mark=false
 spark.hadoop.lakefs.gc.mark_id=<MARK_ID> # Replace <MARK_ID> with the identifier you obtained from a previous mark-only run
 ```
 
-## Considerations
+## Garbage collection notes
```

Comment:
Personal opinion, preferred the previous title

Reply:
noted :)